app/main.py

```diff
@@ -1,3 +1,71 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
```

Comment:
The type hint `int | float` requires Python 3.10 or later. If you need to support earlier versions, use `Union[int, float]` from the `typing` module.